indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/X509SignatureAlgorithm.kt

```diff
@@ -204,6 +257,6 @@ object X509SignatureAlgorithmSerializer : KSerializer<X509SignatureAlgorithm> {
 
     override fun deserialize(decoder: Decoder): X509SignatureAlgorithm {
         val decoded = decoder.decodeString()
-        return X509SignatureAlgorithm.entries.first { it.name == decoded }
+        return X509SignatureAlgorithm.registeredAlgorithms.first { it.name == decoded }
```

Comment:
good thing I have a PR going that gets rid of this ugly hack ;-). fine here, of course